libraries/networking/src/NLPacket.cpp

```diff
@@ -71,8 +71,8 @@ NLPacket::NLPacket(PacketType type, qint64 size, bool isReliable, bool isPartOfM
     writeTypeAndVersion();
 }
 
-NLPacket::NLPacket(std::unique_ptr<Packet> packet) :
-    Packet(std::move(*packet.release()))
```

Comment:
I'm actually just realizing that this was totally a leak.
We were scrubbing the guts of `packet` but never destructing it.